presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -21,10 +21,431 @@
 
 namespace facebook::presto {
 
+#define SYS_CONF_BASE_PROPS(V)                                                \
+  /* Setting this to 'true' makes configs modifiable via server operations.*/ \
+  V(BOOL, kMutableConfig, "mutable-config", false)
+
+#define SYS_CONF_PROPS(V)                                                         \
```

Comment:
Can you retain the original order of properties so that its easier to check that are all still there ?

Reply:
I thought about that but I use alphanumeric and data type as the order at the end. My thought is that it's easier to sort through the properties and I assume people won't care about the grouping and ordering that much at the end of the day.

Let me change back to the original order if you think the original order is meaningful.

BTW, I compared the before and after changes manually, and I am sure I have the whole properties there :-)

Reply:
updated the code to use the original order and rebased the code to solve the conflict.